src/containers/Dashboard/Dashboard.tsx

```diff
@@ -0,0 +1,66 @@
+/* eslint-disable global-require */
+// NOTE: disable global-require for dummy data, will be removed
+// once data is fed top-down
+
+import React from "react";
+import FilterBar from "../FilterBar/FilterBar";
+import { MovieProps } from "../../components/Movie/Movie";
+import MovieList from "../MovieList/MovieList";
+
+import styles from "./Dashboard.module.scss";
+
+// DATA
+
+const genres = ["All", "Documentary", "Comedy", "Horror", "Crime"];
+const sortOptions = ["Release date", "Rating", "Runtime", "Title"];
+
+const movies: MovieProps[] = [
+  {
+    title: "Pulp Fiction",
+    genre: "Action & Adventure",
+    releaseDate: 2004,
+    imgUrl: require("../../images/pulpfiction.jpg"),
+  },
+  {
+    title: "Bohemian Rhapsody",
+    genre: "Drama, Biography, Music",
+    releaseDate: 2003,
+    imgUrl: require("../../images/bohemianrhapsody.jpg"),
+  },
+  {
+    title: "Kill Bill: Vol 2",
+    genre: "Oscar winning Movie",
+    releaseDate: 1994,
+    imgUrl: require("../../images/killbill2.jpg"),
+  },
+  {
+    title: "Avenger: War of Infinity",
+    genre: "Action & Adventure",
+    releaseDate: 2004,
+    imgUrl: require("../../images/avengers.jpg"),
+  },
+  {
+    title: "Inception",
+    genre: "Action & Adventure",
+    releaseDate: 2003,
+    imgUrl: require("../../images/inception.jpg"),
+  },
+  {
+    title: "Resevoir Dogs",
+    genre: "Oscar winning Movie",
+    releaseDate: 1994,
+    imgUrl: require("../../images/resevoirdogs.jpg"),
+  },
+];
+
+const Dashboard = (): JSX.Element => {
+  return (
+    <div className={styles.dashboard}>
+      <FilterBar genres={genres} sortOptions={sortOptions} />
+      <hr className={styles.hr} />
+      <MovieList movies={movies} />
```

Comment:
Additionally, let's simulate the async action of getting the data here.
Under "src/core" folder that you create after one of the above comments consider creating a folder "api" folder that will hold all the specific API requests in your app.
The files under "src/core/api" could be divided into 2 groups: mocked and real.
Use Promise.resolve fro actual async effect and useEffect to get the data.

Reply:
I've implemented a mock api method - used a `.ts` file instead of `.json` as I'm not sure yet how image urls will be handled once integrated with the real API. this way, the url's are resolved in the bundle process.

I wasn't sure if you wanted to see a layered approach to the API (e.g., an `index.ts` which uses mocked or real api calls based on a variable/setting) - so let me know what you think.